### ISSUE TITLE
doc prepare_node()

### ISSUE THREAD

**Issue by nouiz** [Theano/Theano#5217]
We should add it in the extenting theano doc about making a new op.

**Comment by Sentient07** [comment]
I'd like to do this along with the other documentation PR that I'm doing. Is there anyone working on this/assigned to anyone ?

**Comment by nouiz** [comment]
nobody work on this. You already have many PRs going and we have difficulty
to keep up. So maybe make a PR with the doc update you already have before
starting new thing.

thanks

**Comment by Sentient07** [comment]
Yeah, that was what I meant. But #5231 is a superset of my new_gpu_doc branch. So this would be a standalone PR if I make it. Also, I won't be opening one more till I complete the work on RoiPooling
